### PR TITLE
aws_route53_record_set

### PR DESCRIPTION
**Pull request by soumyo13**
Signed-off-by: Soumyodeep Karmakar <soumyo.k13@gmail.com>

### Description

It contains all the singular and plural resources of the above resource.

### Issues Resolved
This is a new resource. There are no issues to be fixed.

### Check List
Please fill box or appropriate ([x]) or mark N/A.
- [x] New functionality includes integration tests/controls
- [x] New Terraform resources
- [x] Documentation provided or updated for resources 
- [ ] All Integration Tests pass
- [x] All Unit Tests pass
- [x] `rake lint` passes
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

**Comment by soumyo13** [comment]
@Nirbhay1997 has done the terraform changes. He will push it once he will push it. Please also update the integration tests with the terraform. Manually I have tested and all are working fine.

File: docs/resources/aws_route53_record_set.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+Ensure that a record exists.
+
+    describe aws_route53_record_set(hosted_zone_id: 'ResourceId', start_record_name: 'DNSName') do
+      it { should exist }
+    end
+
+## Parameters
+
+`hosted_zone_id start_record_name` _(required)_
```
Comment (IanMadd) [inline docs/resources/aws_route53_record_set.md:22]: Is start_record_name required?

Comment (soumyo13) [reply docs/resources/aws_route53_record_set.md:22]: Yes

**Review by IanMadd — APPROVED** [review]

**Review by sathish-progress — APPROVED** [review]
LGTM, please update readme

**Pull request merged**